packages/web-pkg/src/components/Spaces/QuotaModal.vue

```diff
@@ -69,7 +69,11 @@ export default defineComponent({
     }
   },
   mounted() {
-    this.selectedOption = this.spaces[0].spaceQuota.total || 0
+    if (!this.spaces.length) {
+      this.selectedOption = 0
+      return
+    }
+    this.selectedOption = this.spaces[0].spaceQuota?.total || 0
```

Comment:
```suggestion
    this.selectedOption = this.spaces[0]?.spaceQuota?.total || 0
```
With this the early return above is not needed. `this.spaces[0]` is still valid when the array is empty.